### PR TITLE
add nul_creator, nul_subject, and nul_contributor to the image model …

### PR DESCRIPTION
**Pull request by carrickr**
…and Donut vocab

Fixes https://github.com/nulib/next-generation-repository/issues/385

### CODE REVIEW

File: app/models/image.rb [hunk 1]
```diff
@@ -84,6 +84,18 @@ class Image < ActiveFedora::Base
     index.as :stored_searchable, :facetable
   end
 
+  property :nul_creator, predicate: ::Vocab::Donut.has_creator, multiple: true do |index|
```
Comment (kdid) [inline app/models/image.rb]: should this be `::Vocab::Donut.creator`?

Comment (kdid) [inline app/models/image.rb]: should this be `::Vocab::Donut.creator`?

(same for subject and contributor)

Comment (carrickr) [reply app/models/image.rb]: I was going with has, although using underscores, to follow this general concept:

https://samvera.github.io/customize-metadata-model.html#to-add-a-new-multi-value-property

If we have a DONUt way though I'm happy to default to that

Comment (kdid) [reply app/models/image.rb]: Doesn't it need to match what you added to the Donut vocab?

File: app/models/image.rb [hunk 2]
```diff
@@ -84,6 +84,18 @@ class Image < ActiveFedora::Base
...
+    index.as :stored_searchable, :facetable
+  end
+
+  property :nul_subject, predicate: ::Vocab::Donut.has_subject, multiple: true do |index|
```
Comment (kdid) [inline app/models/image.rb]: In the spreadsheet it looks like Jen specified a local predicate for creator and contributor but not for subject, she specified a different one for this

Comment (kdid) [reply app/models/image.rb]: looks like it would be `::RDF::Vocab::DC.subject`

**Pull request merged**